samples-lib/src/main/java/com/esri/arcgismaps/sample/sampleslib/components/LoadingDialog.kt

```diff
@@ -0,0 +1,63 @@
+package com.esri.arcgismaps.sample.sampleslib.components
+
+import android.content.res.Configuration
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Dialog
+import com.esri.arcgismaps.sample.sampleslib.theme.SampleAppTheme
+
```

Comment:
Top level comment on the component would be helpful here:
```suggestion
/**
 * Composable component to display an indeterminate loading dialog along with a [loadingMessage]
 */
```